### PR TITLE
Improve costing calculation for BLS related methods

### PR DESCRIPTION
**Pull request by lrubasze**
## Summary
Improve costing calculation for BLS related methods

more details available in this spreadsheet [here](https://docs.google.com/spreadsheets/d/1rV0KyB7UQrg2tOenbh2MQ1fo9MXwrwPFW4l0_CVO-6o/edit?usp=sharing)

### CODE REVIEW

File: radix-engine/src/system/system_modules/costing/fee_table.rs [hunk 1]
```diff
@@ -399,16 +399,62 @@ impl FeeTable {
...
+        //   Lets round:
+        //    34.42199      -> 35
+        //    2620295.64271 -> 2620296
+        //
+        // Also observed that additional 16850000 instructions are performed
+        // every multiple of 8
+        let mut instructions_cnt = 0u32;
+
+        // P
+        for s in sizes {
```
Comment (iamyulong) [inline radix-engine/src/system/system_modules/costing/fee_table.rs:427]: Remove the `P` comment?

File: radix-engine/src/system/system_modules/costing/fee_table.rs [hunk 2]
```diff
@@ -399,16 +399,62 @@ impl FeeTable {
...
+            instructions_cnt = add(add(instructions_cnt, mul(35, cast(*s))), 2620296);
         }
-        cost
+        let multiplier = cast(sizes.len() / 8);
+
+        instructions_cnt = add(instructions_cnt, mul(multiplier, 16850000));
```
Comment (iamyulong) [inline radix-engine/src/system/system_modules/costing/fee_table.rs:432]: Remove the empty line above?

File: radix-engine/src/system/system_modules/costing/fee_table.rs [hunk 3]
```diff
@@ -399,16 +399,62 @@ impl FeeTable {
...
+            instructions_cnt,
+            match sizes.len() % 8 {
+                1 => 3051556,
+                2 => 5020768,
+                3 => 6990111,
+                4 => 8959454,
+                5 => 10928798,
+                6 => 12898141,
+                7 => 14867484,
+                0 => 0,
```
Comment (iamyulong) [inline radix-engine/src/system/system_modules/costing/fee_table.rs:446]: Can we move `0 => 0` to the first for readability?

**Review by iamyulong — APPROVED** [review]
LGTM, with minor comments.

File: radix-engine/src/system/system_modules/costing/fee_table.rs (hunk 1 above)
Comment (lrubasze) [reply radix-engine/src/system/system_modules/costing/fee_table.rs:426]: Done

File: radix-engine/src/system/system_modules/costing/fee_table.rs (hunk 2 above)
Comment (lrubasze) [reply radix-engine/src/system/system_modules/costing/fee_table.rs:430]: Done

File: radix-engine/src/system/system_modules/costing/fee_table.rs (hunk 3 above)
Comment (lrubasze) [reply radix-engine/src/system/system_modules/costing/fee_table.rs]: Done

**Pull request merged**